blackbox/docs/general/ddl/data-types.rst

```diff
@@ -104,39 +104,38 @@ Columns of type string can also be analyzed. See :ref:`sql_ddl_index_fulltext`.
 Numeric Types
 =============
 
-CrateDB supports a set of numeric types: ``integer``, ``long``, ``short``,
-``double``, ``float`` and ``byte``.
-
-The ``float`` and ``double`` data types are inexact, variable-precision numeric
-types. It means that these types are stored as an approximation. Therefore,
-storage, calculation, and retrieval of the value will not always result in an
-exact representation of the actual floating-point value.
+CrateDB supports a set of the following numeric data types:
+
++----------------------+----------+-----------------------------+-----------------------------+
+| Name                 | Size     | Description                 | Range                       |
++======================+==========+=============================+=============================+
+| ``smallint``         | 2 bytes  | small-range integer         | -32,768 to 32,767           |
++----------------------+----------+-----------------------------+-----------------------------+
+| ``integer``          | 4 bytes  | integer                     | -2^31 to 2^31-1.            |
++----------------------+----------+-----------------------------+-----------------------------+
+| ``bigint``           | 8 bytes  | large-range integer         | -2^63 to 2^63-1             |
++----------------------+----------+-----------------------------+-----------------------------+
+| ``real``             | 4 bytes  | inexact, variable-precision | 6 decimal digits precision  |
++----------------------+----------+-----------------------------+-----------------------------+
+| ``double precision`` | 8 bytes  | inexact, variable-precision | 15 decimal digits precision |
++----------------------+----------+-----------------------------+-----------------------------+
```

Comment:
`char` is missing in that matrix or it links to the wrong section

Reply:
fixed the link, `char` goes to `special character types`